astropy/cosmology/io/tests/test_mapping.py

```diff
@@ -129,6 +129,36 @@ def test_tofrom_mapping_move_tofrom_meta(self, cosmo, to_format, from_format):
         assert got == cosmo  # (Doesn't check metadata)
         assert got.meta["mismatching"] == "will error"
 
+    def test_to_mapping_rename_conflict(self, cosmo, to_format):
+        """Test ``rename`` in ``to_mapping()``."""
+        to_rename = {"name": "name"}
```

Comment:
It would be good to add an entry here that does not cause a conflict so that the error message below could be tested more thoroughly.

Reply:
Done!